main.go

```diff
@@ -40,22 +40,28 @@ func main() {
 		fmt.Printf("%s\n", config.GetDefaultConfig())
 		os.Exit(0)
 	}
+
 	appConfig, err := config.NewAppConfig("lazygit", version, commit, date, buildSource, *debuggingFlag)
 	if err != nil {
 		log.Fatal(err.Error())
 	}
 
-	app, err := app.NewApp(appConfig)
-
-	if err == nil {
-		err = app.Run()
+	appDir := flag.Arg(0)
+	if appDir != "" {
+		err := os.Chdir(appDir)
+		if err != nil {
+			log.Fatal(fmt.Sprintf("Could not change directory to %s", appDir))
```

Comment:
Why not use `log.Fatalf`?

Reply:
I was copying code from below, it's fixed on both occasions now, thanks for the heads up!